### ISSUE TITLE
Travis build failing when af.inverse is being used

### ISSUE THREAD

**Issue by Balavarun5** [QuazarTech/DG_Maxwell#62]
#### Issue
The Travis [build](https://travis-ci.org/QuazarTech/DG_Maxwell/builds/291918310?utm_source=github_status&utm_medium=notification) for the PR #59 is failing for the `test_advection()` test function, This `test_advection()` function uses `af.inverse()`

**Comment by amanabt** [comment]
@Balavarun5 
First, give the full address of the test function, `dg_maxwell/tests/test_waveEqn.py::test_advection`.
Second, PR #59 is for `2D advection solver` only, `test_advection` is a part of `1D advection solver`.
So, this function should not be written in this PR. Please create a different PR and work on it there.

**Comment by Balavarun5** [comment]
This test should be there regardless, It's similar to testing a primary function like 1D A-matrix, volume integral term and the surface term.

**Comment by amanabt** [comment]
I agree that we may build upon the results of this test for 2D advection solver. But that does not explain why this function should be developed in this PR. Let's not make the mistake again of creating huge PR's with many different additions and merging them at once. This only makes the PR confusing.

**Comment by Balavarun5** [comment]
A PR #63 for this issue has been created
